os/drivers/os_api_test/kernel/Make.defs

```diff
@@ -17,7 +17,7 @@
 ############################################################################
 # Include kernel test drivers
 
-ifeq ($(CONFIG_DRIVERS_KERNEL_TEST),y)
```

Comment:
Also, pls search for "kernel test" in whole code and where ever required, pls change it to "os api test"